teachertool/src/components/CatalogModal.tsx

```diff
@@ -6,10 +6,22 @@ import { hideModal } from "../transforms/hideModal";
 import { addCriteriaToRubric } from "../transforms/addCriteriaToRubric";
 import { CatalogCriteria } from "../types/criteria";
 import { getSelectableCatalogCriteria } from "../state/helpers";
+import css from "./styling/CatalogModal.module.scss";
 
-interface IProps {}
+interface CatalogCriteriaDisplayProps {
+    criteria: CatalogCriteria;
+}
+const CatalogCriteriaDisplay: React.FC<CatalogCriteriaDisplayProps> = ({ criteria }) => {
```

Comment:
Would it make sense to factor this component be out for use here in the modal, and also in the criteria table?

Reply:
Maybe. I thought they might diverge again once we have parameterized inputs in the criteria.